app/gateway/changelog.md

```diff
@@ -753,6 +753,48 @@ This should be a 400 because the configuration is invalid.
 * When the OpenID Connect (OIDC) plugin is configured to reference HashiCorp Vault in the `config.client_secret` field (for example, `{vault://hcv/clientSecret}`),
 it does not look up the secret correctly.
 
+## 3.2.2.4
+**Release Date** 2023/08/21
+
+### Breaking changes and deprecations
+
+* **Ubuntu 18.04 support removed**: Support for running Kong Gateway on Ubuntu 18.04 ("Bionic") is now deprecated,
+as [Standard Support for Ubuntu 18.04 has ended as of June 2023](https://wiki.ubuntu.com/Releases).
+Starting with Kong Gateway 3.2.2.4, Kong is not building new Ubuntu 18.04
+images or packages, and Kong will not test package installation on Ubuntu 18.04.
+
+    If you need to install Kong Gateway on Ubuntu 18.04, see the documentation for
+    [previous versions](/gateway/3.1.x/install/linux/ubuntu/).
```

Comment:
Hmm. I actually don't really know how to handle this 🤔 I _think_ the deprecation is going to be applicable as far back as 3.1.x, since we're still building patches for that. So this statement won't be accurate once the next 3.1.x patch is out.

Maybe the line needs to be something like this:

```suggestion
    If you need to install Kong Gateway on Ubuntu 18.04, substitute a previous 3.2.x 
    patch version in the [installation instructions](/gateway/3.2.x/install/linux/ubuntu/).
```
What do you think?

Reply:
I think that makes sense to me!